src/mbgl/map/map.cpp

```diff
@@ -54,6 +56,7 @@ class Map::Impl : public style::Observer {
 
     RenderState renderState = RenderState::Never;
     Transform transform;
+    std::unique_ptr<CameraOptions> initialCameraOptions;
```

Comment:
that is correct, I changed it with `optional` and using `std::experimental::nullopt` instead of `nullptr` as a reset value.